lib/code_corps_web/plugs/data_to_attributes.ex

```diff
@@ -0,0 +1,21 @@
+defmodule CodeCorpsWeb.Plug.DataToAttributes do
+  @moduledoc """
+  Puts authenticated Guardian user into conn.assigns[:current_user]
```

Comment:
@begedin Is this moduledoc correct?

Reply:
@snewcomer It was absolutely not. I'll do a quick correction PR

Reply:
Fixed in #884